storage/kv_test.go

```diff
@@ -773,8 +770,8 @@ func TestWatchableKVWatch(t *testing.T) {
 		if !reflect.DeepEqual(ev, wev) {
 			t.Errorf("watched event = %+v, want %+v", ev, wev)
 		}
-	case <-time.After(time.Second):
-		t.Fatalf("failed to watch the event")
+	case <-time.After(5 * time.Second): // wait 5 seconds for CI with slow IO
```

Comment:
I feel it is just because CPU is slow so the routine is not able to switch around.